src/generalizedlinearmixedmodel.jl

```diff
@@ -793,7 +794,7 @@ function updateη!(m::GeneralizedLinearMixedModel{T}) where {T}
     b = m.b
     u = m.u
     reterms = m.LMM.reterms
-    mul!(η, modelmatrix(m), m.β)
+    mul!(η, view(modelmatrix(m), :, 1:(m.LMM.feterm.rank)), m.β)
```

Comment:
This is the same but IMO is easier to read/understand at a glance:
```suggestion
    mul!(η, view(modelmatrix(m), :, 1:rank(m)), m.β)
```